<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popFrmArithmetics</class>
 <widget class="QFrame" name="popFrmArithmetics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label_6">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Shot</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_7">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Channel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_8">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Multiplier
(default 1)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbxShotNum">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="cbxChannel"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="ledMultiplier">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="cbxOperator_1">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>/</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbxShotNum_2">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="cbxChannel_2"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="ledMultiplier_2">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="cbxOperator_2">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>/</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cbxShotNum_3">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="cbxChannel_3"/>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="ledMultiplier_3">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="cbxOperator_3">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>/</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbxShotNum_4">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="cbxChannel_4"/>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="ledMultiplier_4">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="cbxOperator_4">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>/</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cbxShotNum_5">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="cbxChannel_5"/>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="ledMultiplier_5">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QPushButton" name="btnClear">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="2">
    <widget class="QPushButton" name="btnCalculate">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
